pkg/placement/controller.go

```diff
@@ -413,13 +440,24 @@ func (c *Controller) reconcile(ctx context.Context, key util.Key) error {
 		if err := c.handleCRD(obj); err != nil {
 			return err
 		}
+	} else if util.IsPlacementDecision(obj) {
+		if err := c.handlePlacementDecision(obj); err != nil {
+			return err
+		}
+		return nil
```

Comment:
This could be simpler.
```suggestion
		return c.handlePlacementDecision(obj)
```